### PR TITLE
feat: documents page (new IA)

### PR DESCRIPTION
**Pull request by imklau**
## Description 📝

This PR introduces new documents pages: `/umowy/` and `/ankiety/`.

## Related Issues 🔗

JIRA issue: [FSU-394](https://klau.atlassian.net/browse/FSU-394)

### CODE REVIEW

File: packages/frontend/app/templates/Agreements/Agreements.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        <Container size="medium">
+          <Styled.DocumentContainer>
+            <Styled.HeaderContainer>
+              <Typography variant="h3" as="h2">
+                Niezbędne dokumenty
+              </Typography>
+              <Typography color={theme.colors.gray[500]} variant="bodyTitle">
+                Zostałeś zaakceptowany do adopcji lub wybrany, aby zostać domem
+                tymczasowym? Poniższe umowy są dla Ciebie.
```
Comment (paulgrym) [inline packages/frontend/app/templates/Agreements/Agreements.jsx:44]: in the design width of this paragraph is limited to ~662px

File: packages/frontend/app/templates/Agreements/components/Hero/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <Styled.Wrapper>
+        <Styled.TextWrapper>
+          <TextBanner
+            size="large"
+            heading="Umowy adopcyjne"
+            tabletLayout="left"
+            mobileLayout="left"
+          >
+            Tutaj znajdziesz umowy adopcyjne i niezbędne informacje, aby je
+            wysłać do fundacji.
```
Comment (paulgrym) [inline packages/frontend/app/templates/Agreements/components/Hero/Hero.jsx:20]:
```suggestion
            Tutaj znajdziesz umowy adopcyjne i niezbędne informacje, żeby dokończyć adopcje wybranego czworonoga.
```

Comment (paulgrym) [reply packages/frontend/app/templates/Agreements/components/Hero/Hero.jsx:20]: There is new text in the design after your suggestions :)

**Review by paulgrym — APPROVED** [review]
Great job!

File: packages/frontend/app/templates/Documents/components/DocumentsList/DocumentsList.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      </a>`,
+      `poczekaj na odpowiedź od nas`,
+    ],
+    link: `${globalData.surveys.temporaryHome}`,
+    asset: <TempHomeSurvey />,
+  },
+]
+
+const DocumentsTemplate = () =>
+  data.map((document) => (
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Documents/components/DocumentsList/DocumentsList.jsx:45]: I would dectructure it :D

**Review by AmandaKr — APPROVED** [review]
🏅

**Pull request merged**